includes/features/stanford_metatag_nobots.feature

```diff
@@ -1,9 +1,10 @@
+@stanford
 Feature: Stanford MetaTag NoBots
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure that the Stanford MetaTag NoBots module is working properly
 
-  @api
+  @api @safe @deploy
```

Comment:
Oh good call. The test does enable a module. I would not want to label a test 'safe' if it enables anything. I suspect `@deploy` would be fine as if the test fails chances are you will have to rebuild the site anyhow.